<?xml version="1.0"?>

<launch>
    <node pkg="ftm_planning" type="bluetoothConnecter.py" name="bluetoothConnecter" respawn="true" output = "screen">
    </node>
    <node pkg="rosserial_python" type="serial_node.py" name="serial_node" output ="screen">
        <param name="port" type="string" value="/dev/ttyACM0"/>
        <param name="baud" type="int" value="57600"/>
    </node>
</launch>
